### ISSUE TITLE
Emails: Improve the interface to improve discoverability & scalability

### ISSUE THREAD

**Issue by ericnicolaas** [Charitable/Charitable#705]
The main emails settings area in Charitable is getting unwieldy, as more and more emails are being added (I'm about to add 2-4 more in Ambassadors). Here's a screenshot of what it looks like right now:

![Screenshot 2019-09-19 09 21 31](https://user-images.githubusercontent.com/676582/65195582-fa549780-dabe-11e9-8bee-a71521b6aabc.png)

There is a lot of capability & flexibility here, but with so many listed and sharing similar names, I feel like it's easy to gloss over this, or feel overwhelmed.

One alternative that I think might work is to organize the list of emails by trigger events. So for example:

_When a donation is completed (marked as Paid)_

- **Donation Receipt** sent to the Donor
- **New Donation Notification** sent to the Admin
- **Donation Notification** sent to the Campaign Creator

_When a campaign is submitted_

- **New Campaign** sent to the Campaign Creator
- **New Campaign Notification** sent to the Admin

...

Obviously this would be laid out a little more attractively, with a button to enable/disable the email, and a button to click to handle the email's settings, but hopefully it illustrates the concept I'm proposing.

**Comment by helgatheviking** [comment]
You could also organize by email recipient.  so: admin, donor, campaign creator, etc

**Comment by ericnicolaas** [comment]
@helgatheviking Yeah, I was thinking about that approach too. I just wasn't sure if that was quite as intuitive, though it's really just a different way of looking at it I guess. 

I think organizing by trigger is a bit of a value-add to in the sense that it highlights the trigger for different emails. That could also help us explain some of the finer details of when emails are sent. For example, it can be a bit confusing to distinguish between the Admin Recurring Notification & Admin Renewal Notification emails, so grouping them by trigger would help clarify that.

**Comment by helgatheviking** [comment]
You're probably right. Organizing by action reveals the most information at a glance.
